UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -108,11 +108,11 @@
 
 	<body>
 		<section id="abstract">
-			<p>Reading a publication is a very personal experience. For most of us this is routine, and give little
+			<p>Reading a publication is a very personal experience. For most of us this is routine, and we give little
```

Comment:
The usual guidance for specifications is to avoid personal pronouns.

While this is only a CG note, it might still be worth following the practice. So for sentences like this, you could establish you're talking about users: "For most users, this is routine and they give little consideration to how they obtain the title before they read it.

You might want to tackle this in a separate pull request, though.